utils/vlog/vlog_to_pbtype.py

```diff
@@ -114,6 +114,7 @@
 def mod_pb_name(mod):
     """Convert a Verilog module to a pb_type name in the format documented here:
     https://github.com/SymbiFlow/symbiflow-arch-defs/#names"""
+    return mod.name
```

Comment:
What's going on here?

Reply:
this most probably is making CI fail as the pb_type name will be wrongly translated, I will remove this `return` and check if that's the case. Probably it's a leftover from the WIP